public/assets/css/main.css

```diff
@@ -78,6 +78,31 @@ figcaption {
 	box-shadow: rgb(157, 18, 60) 0 0 .2em;
 }
 
+ul {
+	list-style-type: none;
+	font-size: 80%;
+}
+
+ul li {
```

Comment:
display: inline; to have the links one after the other

Reply:
`display: inline;` didn't work but links set one after the other using `flex` (thus, similar css to the on sections).